Maca aporte2

Macarena12345 opened the pull request:
<!-- Saludos, si usted esta leyendo esto, significa que quiere colaborar con el proyecto aportando quizás mejoras o información que encontró mientras utilizo el mismo, es por eso que queremos agradecerle que se tome su tiempo por hacer lo antes mencionado. -->

<!-- 
	Utilice la siguiente forma de títulos-
•	Si es una mejora en la base de datos [DB] titulo
•	Si es una mejora de estilos [CSS] titulo
•	Si es una mejora en el código [HTML] titulo
Si utiliza varias mejoras, no especifique nada, solo el titulo.

--> HTML

<!-- Necesitamos el que titulo del tema sea lo mas representativo posible. -->

##### ¿QUÉ CAMBIO PROPONE?:
<!--
	Agregue un pequeño texto que explique cual es el bug que desea reportar o cual es la situación de mejora que le gustaría que apliquemos. 
-->
El cambio que propongo es de agregar la ubicacion, para esto puse con en un subtitulo el barrio, calle y telefono, luego abajo de esto coloque un mapa de google maps que me dice la direccion del taller.



##### NOTAS EXTRA:
<!--
Cualquier información que pueda ayudar a los desarrolladores a identificar y solucionar el problema debe ser puesta aquí.
-->

##### BRANCH(ES):
<!-- 
Especifique la(s) rama(s) afectada(s) por este problema: master, 0.x, 1.x, u otra rama. 
Reemplace el valor de master por la rama correspondiente.
-->

master



##### OTRAS PERSONALIZACIONES:
<!-- 
En caso de que se produzca alguna mejora en el código, avísennos, caso contrario, no es necesario que usted escriba nada aquí. (solo para programadores)
-->




<!-- ------------------------- THE END ------------------------------
 Gracias nuevamente por su colaboración, atenderemos su tema lo antes posible. 
-->

Macarena12345 committed:
Seccion Contacto

En esta seccion llamada contaacto.html es una parte de la pagina donde me encuentro
con un formulario que contiene el nombre, apellido, correo electronico, un textarea
para que el cliente ingrese los inconvenientes que tiene y tambien un radio que nos
sirve para saber si tiene aseguradora o  es particular. por ultimo cuenta con un boton
para que el cliente nos envie toda la informacion ingresada.

Macarena12345 committed:
Agregando estilo a la pagina

Este en un css en comun, para colocarle un estilo a todas las paginas de nuestro
sitio. En esta seccion agrege el estilo de la seccion contacto y  la seccion
sobre nosotros. Este es un archivo donde cada uno puede ir haciendo modificaciones
de cada seccion de nuestro sitio.

Macarena12345 committed:
Seccion contacto.html

En esta seccion agregue la opcion para que el usuario me ingrese toda la
informacion necesaria para nosotros, como puede ser el nombre, apellido,
correo electronico, un textarea donde el usuario puede contarnos sobre
su problema, y por ultimo cuenta con un boton donde me envia la informacion.

Macarena12345 committed:
Agregando estilo

Agregue un archivo llamado style.css donde vamos a colocar todo el estilo de
las paginas de nuestro sitio web.

Macarena12345 committed:
Agregando un mapa

Agregue un mapa de google maps a la pagina contacto.html, donde me muestra la
ubicacion del taller.

Macarena12345 committed:
Merge branch 'master' of https://github.com/RainTrap/Trabajo-Practico-3 into maca_aporte2

Macarena12345 committed:
Modificando el Style.css

La modificacion que hice fue agregarle estilo a la ubicacion.

RainTrap reviewed (changes requested):
Cambiar lenguaje a "es". Cambiar meta name ya que no es el indicado. Cambiar title de la seccion head ya que no es el indicado para la pagina contacto. Cambiar la linea 68 para que diga: "B° Valentina Sur / Virgilio Amarante Mza K / Tel: 2994698624". En el archivo style.css agregar un espacio al final del archivo.

Macarena12345 committed:
Modificando contacto.html

Cambie el lenguaje a "es". Cambie el meta name por el conrrespondiente
Cambie el title colocandole Contacto. Y por ultimo en la linea 68
agregue lo solicitado.

Macarena12345 committed:
Modificando el style.css

Agregue, como lo solicitaba, un espacio al final de la pagina.

RainTrap reviewed (approved):
Gracias por realizar los cambios solicitados. El código esta bien.
